go/libkb/rpc_exim.go

```diff
@@ -1302,12 +1304,12 @@ func (u *User) ExportToUPKV2AllIncarnations() (*keybase1.UserPlusKeysV2AllIncarn
 				return nil, fmt.Errorf("Tried to export empty subchain for uid %s username %s", u.GetUID(), u.GetName())
 			}
 			cki := subchain[len(subchain)-1].cki
-			pastIncarnations = append(pastIncarnations, cki.exportUPKV2Incarnation(uid, name, subchain[0].GetSeqno(), kf))
+			pastIncarnations = append(pastIncarnations, cki.exportUPKV2Incarnation(uid, name, subchain[0].GetSeqno(), kf, status))
```

Comment:
I wonder if these past incarnations should get `SCAccountReset` or something instead of whatever the current incarnation's status is? It might be weird to have a status on past incarnations at all, since the concept of status is only meaningful for the current incarnation. But maybe more convenient that sticking it on the all-incarnations struct, since there are places where we pass around a single incarnation that's assumed to be current?

Reply:
Hmm, I was wondering about that when hacking this out.  Would you want that  for previous incarnations of reset accounts too?

Reply:
I guess code that's handling a past incarnation of a user should be well aware that that's what it's doing, and so should never really look at the "status"? So I don't feel strongly.

Maybe in a perfect world we would have a slightly different type for past incarnations, similar to Current but missing stuff like status and username and uid. Just to make it statically impossible to mistake it for a current incarnation somehow. Not something I think we should plumb through right now though.